packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -104,11 +104,13 @@ void main() {
     const Color customColor1 = const Color(0xcafefeed);
     const Color customColor2 = const Color(0xdeadbeef);
     const Color customColor3 = const Color(0xdecaface);
+    const Color customColor4 = const Color(0xfeedcafe);
 
     final SliderThemeData sliderTheme = new SliderThemeData.fromPrimaryColors(
       primaryColor: customColor1,
       primaryColorDark: customColor2,
       primaryColorLight: customColor3,
+      valueIndicatorTextStyle: const TextStyle(color: customColor4),
```

Comment:
case in point, if you did this, without doing the merge thing for the font, it would imply all defaults for the text style except the colour, which is unlikely what anyone wants -- i'd expect this to just override the current color.

Reply:
Yep, makes sense.  I'll add the merge.

Reply:
So, the way I have it now, there isn't a need for a "merge" anymore because the default happens in the ThemeData where the default sliderTheme is created.  Then it's up to the developer to get their (required) TextStyle from a source (fallback, ambient, whatever) theme in the right way (by merging two, or using copyWith).

The question is: does it make sense to use merge() inside of SliderThemeData's copyWith function for the text style?  I think the answer is "no", since that wouldn't be what they expected (they'd expect a replacement), but you're sort of making me wonder with this question.